litentry/core/stf-task/receiver/src/stf_task_receiver.rs

```diff
@@ -85,13 +85,26 @@ where
 				)
 				.map_err(|e| Error::OtherError(format!("error verify web3: {:?}", e)))?;
 
-				let c = context.create_verify_identity_trusted_call(
+				let c = context.create_identity_verify_trusted_call(
 					request.who,
 					request.identity,
 					request.bn,
 				)?;
 				let _ = context.submit_trusted_call(&c)?;
 			},
+			RequestType::RulesetVerification(request) =>
+				for identity in request.vec_identity {
+					let result = lc_ruleset_build::ruleset_verify(
+						request.who.clone(),
+						identity,
+						request.ruleset.clone(),
+					)
+					.map_err(|e| Error::RulesetError(format!("error verify ruleset: {:?}", e)));
+
+					if result.is_ok() {
+						break
```

Comment:
Let's add a comment here, something like:
```
//TODO VC generation and pass back
```
I notice there's one inside `ruleset2_verification`, but putting it here clarifies the code structure that "it should only be done when `result.is_ok()`"

Reply:
Actually until here (result is Ok returned), VC generation is already done.